app/src/androidTest/java/com/github/sdpteam15/polyevents/fakedatabase/FakeDatabase.kt

```diff
@@ -51,23 +51,23 @@ object FakeDatabase : DatabaseInterface {
         collection: DatabaseConstant.CollectionConstant,
         adapter: AdapterToDocumentInterface<in T>
     ): Observable<String> {
-        TODO("Not yet implemented")
+        throw UnsupportedOperationException("not supported for Fakedatabase")
     }
 
     override fun <T : Any> addEntity(
         element: T,
         collection: DatabaseConstant.CollectionConstant,
         adapter: AdapterToDocumentInterface<in T>
     ): Observable<Boolean> {
-        TODO("Not yet implemented")
+        throw UnsupportedOperationException("not supported for Fakedatabase")
     }
 
```

Comment:
Do we still need this class ? Mock are superiors for the rest of the team

Reply:
as discussed I think they are still very powerfull for UI tests and